max_training_framework/setup/setup_functions.py

```diff
@@ -46,7 +46,10 @@ def get_instance_details(self, response, which_resource,
                     enumerate(value['plan_history'], start=1):
                 for key, values in \
                         enumerate(self.data[which_resource].items()):
-                    if plan['resource_plan_id'] == values[1]:
+                    if plan['resource_plan_id'] == values[1] \
+                            and (value['region_id'] not in
+                                 ['eu-de', 'jp-tok'] or
```

Comment:
It is safer to test for `['eu-gb', 'us-south']` like it was done in `service_handling.py`. If a new data center is opened and it does not support WML training this code would fail to catch the error condition.